.github/workflows/holochain-build-and-test.yml

```diff
@@ -28,9 +28,9 @@ jobs:
       matrix:
         cmd:
           # ensure functionality on multiple platforms
-          - pkgs:
-              - build-holonix-tests-integration
-            extra_arg: "--override-input versions ./versions/weekly --override-input holochain ${{ inputs.repo_path }}"
+          # - pkgs:
```

Comment:
Choosing to comment rather than delete for now because it's possible the release automation doesn't work with the changes I've made and this is the only way we'd have caught that in development so... we'll see